symphony/assets/js/src/symphony.notify.js

```diff
@@ -69,6 +70,13 @@
 				storage = (window.localStorage[settings.storage]) ? $.parseJSON(window.localStorage[settings.storage]) : [];
 			}
 
+			// Dimmed success transition
+			if (!!type && !!~type.indexOf('success')) {
```

Comment:
Seems that it's equivalent to `ToBoolean`, see: http://stackoverflow.com/questions/10467475/double-negation-in-javascript-what-is-the-purpose